products/llbuildSwift/BuildSystemBindings.swift

```diff
@@ -264,9 +270,10 @@ public struct CommandExtendedResult {
         }
     }
 
-    public init(result: CommandResult, exitStatus: Int32) {
+    public init(result: CommandResult, exitStatus: Int32, pid: Int32) {
```

Comment:
If we were to keep this constructor, that pid arg should probably be a pid_t?.   However, I think this is left over from a transitionary step and is no longer used.  Probably cleaner to delete it rather than change its signature.

Reply:
You are right, we should delete it. The command extended result should not be initialized with information not coming from an `llb_buildsystem_command_extended_result_t` instance.

Reply:
Sorry, I had to add this back. It does make sense to open up this initializer since the client could define its own `ExternalCommand`s which should be able to produce extended results on them own. I also opened up the init of CommandMetrics since external commands should also be able to provide metrics over this mechanism.